android/src/main/java/com/swmansion/rnscreens/ScreenFragment.java

```diff
@@ -39,6 +46,10 @@ protected static View recycleView(View view) {
   }
 
   protected Screen mScreenView;
+  private boolean mIsSendingProgress;
+  private Screen mAboveScreen;
```

Comment:
I'd be hesitant of storing reference to other screens here especially if we want to get reference to a screen that's directly before this one in the container. Note that this order can bemodified and that other screen can be removed (e.g. as a result of JS update) from the container without any notice. In this case we'd keep a reference to an orphaned view here

Reply:
We could make `mAboveScreen` null at the end of the transition so it is only alive for the lifetime of it.

Reply:
^ done